File: rx-assist/src/main/java/com/kolyall/rxassist/repositories/AssistRepository.java
package com.kolyall.rxassist.repositories;

import com.assist.api.models.AssistTransactionRequest;

import ru.assisttech.sdk.AssistResult;
import rx.Observable;

/**
 * Created by User on 26.04.2017.
 */

public interface AssistRepository {
    Observable<AssistResult> recurrentCardPayment(String billNumber, AssistTransactionRequest transaction);
}

File: rx-assist/src/main/java/com/kolyall/rxassist/repositories/AssistRepository.kt
package com.kolyall.rxassist.repositories

import com.kolyall.rxassist.models.AssistTransactionRequest

import io.reactivex.Observable
import ru.assisttech.sdk.AssistResult

/**
 * Created by User on 26.04.2017.
 */

interface AssistRepository {
    fun recurrentCardPayment(billNumber: String, transaction: AssistTransactionRequest): Observable<AssistResult>
}
